src/lib/constants.ts

```diff
@@ -9,25 +10,46 @@ export const exitWithMessage = (message: string): void => {
   process.exit(0);
 };
 
+/**
```

Comment:
This file is getting convoluted. I'd move this to another file so it lives on its own and keep this file only for strings